src/components/menu/menu.spec.ts

```diff
@@ -13,17 +13,18 @@ import { MenuModule } from './menu.module';
                 type="button"
                 id="menu-item-1"
                 uxMenuItem
+                (click)="onClick()"
                 [disabled]="disabled"
                 (activate)="onActivate($event)">
                 Item One
             </button>
             <ux-menu-divider></ux-menu-divider>
             <button
                 type="button"
-                id="menu-item-disabled"
+                id="sub-menu-item"
```

Comment:
I think this `id` is misleading, this is an item in the top level menu, not the sub-menu as the `id` suggests. I don't think this id is ever used either?

Reply:
That's an oversight, it needs removed it was for a previous test that had to be reconfigured